src/components/validations/with-validation.spec.js

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
 import { shallow } from 'enzyme';
 import PropTypes from 'prop-types';
+import _capitalize from 'lodash/capitalize';
```

Comment:
We're trying to remove lodash as a dependency, we have a `titleCase` util that achieves this in:
`src/utils/helpers/text/text.js`

Reply:
Updated in commit 932b74d3c